core/src/main/java/com/datastax/oss/driver/internal/core/loadbalancing/LatencySensitiveLoadBalancingPolicy.java

```diff
@@ -0,0 +1,275 @@
+/*
```

Comment:
I still need to give this a more complete review but at a minimum this class needs to go in the [examples directory](https://github.com/apache/cassandra-java-driver/tree/4.x/examples/src/main/java/com/datastax/oss/driver/examples) rather than inline with the default LBP like this.  In 4.x the default LBP is intended to be the primary LBP used in nearly all situations; adding another LBP into the same package will significantly confuse that messaging.